README.md

```diff
@@ -1,34 +1,43 @@
 # Adyen Postman
 
+**TL;DR : Visit our always up to date [Postman workspace](https://www.postman.com/adyendev/) to access all of our APIs.**
 
 This repository contains declaration files in the [Postman](https://postman.com/) format. The files are automatically generated based on the latest [adyen-openapi](https://github.com/adyen/adyen-openapi) definition files.
 They are then uploaded on our [AdyenDev](https://www.postman.com/adyendev/) Postman space and you can fork it from there. You can also directly pick the `.json` files available here and upload them if you prefer.
 
-## Usage
+## Using the [AdyenDev](https://www.postman.com/adyendev) workspace
 
-The preferred way to use these files is to fork the [Adyen APIs](https://adyendev.postman.co/workspace/Adyen-APIs~a8d63f9f-cfc7-4810-90c5-9e0c60030d3e) Postman workspace. This way you will be notified of new versions and changes.
-However, in case you want to experiment, or simply want to keep your experiments private, you can also take any of the json files in this repository and upload them yourself.
+The preferred way to use these files is to fork the [Adyen APIs](https://adyendev.postman.co/workspace/Adyen-APIs~a8d63f9f-cfc7-4810-90c5-9e0c60030d3e) Postman workspace. 
 
-![Importing a json file as a Postman collection](./images/import-postman.gif)
+![Creating a fork for a Postman collection](./images/create-fork.gif)
+
+
+This way you will be notified of new versions and changes per email and in the UI directly.
+
+![Receiving the email of a collection changed](./images/postman-email.png)
 
+## Importing the JSON files
 
-## Contributing
+However, in case you want to experiment, want to keep your experiments private or want to use an older version of any of our APIS, you can also take any of the json files in this repository and upload them yourself.
```

Comment:
I would remove `want to keep your experiments private` as forks can also be private.
Change `APIS` to `APIs`